src/sidebar/store/modules/session.ts

```diff
@@ -66,14 +80,32 @@ function isLoggedIn(state: State) {
   return state.profile.userid !== null;
 }
 
+/**
+ * Return the effective set of feature flags. This combines feature flags from
+ * the profile with those from other sources.
+ */
+const features = createSelector(
+  (state: State) => state.profile,
+  (state: State) => state.features,
+  (profile: Profile, features: string[]): Record<string, boolean> => {
+    const combinedFeatures = { ...profile.features };
+    if (features) {
+      for (const feat of features) {
+        combinedFeatures[feat] = true;
+      }
+    }
```

Comment:
Maybe we can avoid this if by initializing features as an empty array.

```suggestion
  (profile: Profile, features: string[] = []): Record<string, boolean> => {
    const combinedFeatures = { ...profile.features };
    for (const feat of features) {
      combinedFeatures[feat] = true;
    }
```

Reply:
`features` here is an array, so this check does nothing. We have a lint check that is supposed to catch mistakes like this. Apparently that didn't work. `features` is initialized to an empty array in the state initializer.